core/layouts/harris/harris.info

```diff
@@ -0,0 +1,19 @@
+name = Harris
+version = BACKDROP_VERSION
+backdrop = 1.x
+type = layout
+
+; Specify regions for this layout.
+regions[header]     = Header
+regions[top]        = Top
+regions[content]    = Content
+regions[sidebar]    = First Sidebar
+regions[sidebar2]   = Second Sidebar
```

Comment:
After discussing with @jenlampton and @biolithic, we decided that the new region names are indeed better. But we still need to accommodate for moving between old and new layouts, both in the D7 upgrade path and in the Layout UI.